docs/installing_deis/aws.rst

```diff
@@ -172,6 +172,21 @@ Install Deis Platform
 Now that you've finished provisioning a cluster, please refer to :ref:`install_deis_platform` to
 start installing the platform.
 
+CloudFormation Updates
+----------------------
+
+To use CloudFormation to perform update operations to your stack, there is another script: 
+`update_ec2_cluster.sh`_. Depending on the parameters that you have changed, CloudFormation
+may replace the EC2 instances in your stack.
+
+The following parameters can be changed without replacing any resources:
+
+- ``ClusterSize`` - Number of nodes in the cluster. This may launch new instances or terminate
+  existing ones, but service will not be interrupted.
```

Comment:
I think this is a bit misleading. If you're scaling down, it may interrupt service. If a component/application was running on an instance that was terminated, it will have to be rebalanced onto another node which will cause some downtime.